src/api/app/services/consistency_check_job_service/project_meta_checker.rb

```diff
@@ -28,8 +28,8 @@ def backend_meta
     end
 
     # transform hash to array, compare it and transform it back to hash
-    def hash_diff(a, b)
-      difference = a.size > b.size ? a.to_a - b.to_a : b.to_a - a.to_a
+    def hash_diff(array_a, array_b)
+      difference = array_a.size > array_b.size ? array_a.to_a - array_b.to_a : array_b.to_a - array_a.to_a
```

Comment:
I really don't see any reasoning why change it. For one-liner, same rule as for blocks should be used.. to tricky rubocop didn't make the code cleaner.

Reply:
The only difference is the naming of the variables, right?

Reply:
yes